client/src/components/Friends/Friends.jsx

```diff
@@ -4,8 +4,54 @@ import './Friends.scss';
 
 // Don't change this <main> wrapper, this tag is used in App.scss
 export default function Friends() {
+
+  const friendsList = [
+    {
+      id: 1,
+      name: 'John',
+      age: 30,
+      gender: 'male',
+      city: 'New York',
+      interests: ['reading', 'hiking', 'traveling'],
+      avatar: 'https://partyanimals.com/static/avatars-02.png',
+    },
+    {
+      id: 2,
+      name: 'Sarah',
+      age: 25,
+      gender: 'female',
+      city: 'San Francisco',
+      interests: ['yoga', 'music', 'painting'],
```

Comment:
Is this how we're going to want to store interest in the database? Also have we established what all will be saved with the user?